baiji/util/test_tempfile.py

```diff
@@ -1,8 +1,8 @@
 import os
 import unittest
+from test.test_support import EnvironmentVarGuard
```

Comment:
We should probably replace this usage with `BackupEnvMixin`, as the `test` package is python internal and not a stable api... But it doesn't have to happen now.